contrib/fedora/cjdns.genconf.patch

```diff
@@ -0,0 +1,20 @@
+diff -up ./client/cjdroute2.c.genconf ./client/cjdroute2.c
+--- ./client/cjdroute2.c.genconf	2015-09-26 23:08:20.832843690 -0400
++++ ./client/cjdroute2.c	2015-09-26 23:13:05.288687831 -0400
+@@ -306,14 +306,14 @@ static int genconf(struct Random* rand,
+            "        // and ETHInterface will be unable to hot-add new interfaces\n"
+            "        // Use { \"setuser\": 0 } to disable.\n"
+            "        // Default: enabled with keepNetAdmin\n"
+-           "        { \"setuser\": \"nobody\", \"keepNetAdmin\": 1 },\n"
++           "        { \"setuser\": \"cjdns\", \"keepNetAdmin\": 1 },\n"
```

Comment:
Adding those flags would reduce the number of patch files needed to be applied for a current fedora build.  If you think other systems will find them useful, then I can work on a change to add them.  I believe I did the "pull request" thing correctly this time!  This patch file will still be needed, however, so that 17.1 and below can be built.  (That "reproducible build" requirement.)  

If having these fedora details in cjdns at all is clutter (even confined to the contrib/fedora directory), then maybe they should be removed.  I envisioned also having someone contribute a contrib/debian, etc.  Maybe it would be clearer to have a contrib/packaging/{fedora,debian,ubuntu,etc}.